src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add(value) {
+      result += value;
+    },
+
+    subtract(value) {
+      result -= value;
+    },
+
+    multiply(value) {
+      result *= value;
+    },
+
+    divide(value) {
+      if (value !== 0) {
+        result /= value;
+      }
+    },
+
+    reset(value) {
```

Comment:
[NAMING] - The parameter `value` in the `reset` method is not used and should be removed as it may cause confusion.